Google.Api.Generator/ProtoUtils/ProtoExtensions.cs

```diff
@@ -25,16 +25,20 @@ namespace Google.Api.Generator.ProtoUtils
 {
     internal static class ProtoExtensions
     {
+        private static T ReadOption<T>(IDescriptor desc, string optionName)
+        {
+            // There is no support for reading various standard options; so use reflection.
```

Comment:
Eek, that's pretty nasty. We should look at fixing that in Google.Protobuf...

Reply:
one day...